### DISCUSSION TITLE
Long dag run vs short task

### DISCUSSION THREAD

**Discussion by clintf1982** [apache/airflow#36137] [General]
<img width="468" alt="image" src="https://github.com/apache/airflow/assets/102470463/c7b43a42-cac2-46e2-8edd-7e867396f35f">

I have a very simple dag that its task takes less than a second to run but it seems that it takes a long time until it is queued, as you can see sometimes almost 5 minutes. There DAGs that are scheduled to run every 15 minutes.
I was wondering if its normal that it would take so long to queue the task ?

**Comment by potiuk** [comment]
Depends on many, many factors.

You can check the list to start with: https://airflow.apache.org/docs/apache-airflow/stable/faq.html#why-is-task-not-getting-scheduled
